### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by joel-becker** [evanjflack/bacon#58]
Hi @evanjflack @EdJeeOnGitHub -- how does your testing workflow work?

My understanding was that [`.travis.yml`](https://github.com/evanjflack/bacon/blob/master/.travis.yml) would automatically run your [testthat tests](https://github.com/evanjflack/bacondecomp/tree/master/tests) upon pushes to master.

But this understanding seems incomplete. The former file seems to quote [this script](https://github.com/evanjflack/bacon/blob/master/codecov.yml), and [other files seem to perform other checks](https://github.com/evanjflack/bacondecomp/blob/master/.github/workflows/rcmdcheck.yml). Could you help my understand what's going on here?

**Comment by evanjflack** [comment]
There are two different testing workflows. Travis executes everything in the testthat folder, the second file you reference is the R CMD check, which doesn’t run the testthat things, but rather looks to see if you have functions from other packages that aren’t imported, and other R package specific things.

**Comment by evanjflack** [comment]
@joel-becker check this out 

http://r-pkgs.had.co.nz/check.html

Ed Edit: R CMD Check will also run all the tests in `tests/` too.

**Comment by EdJeeOnGitHub** [comment]
Also we're using both github actions and travis to test stuff which is a little overkill which might add to the confusion.

**Comment by joel-becker** [comment]
Thank you for all the above, both!

To be sure I understand, does Ed's comment + edit mean that:
- Travis is running testthat tests and R CMD checks
- Github actions is running R CMD checks

Also Evan, I'm not sure from your comment what `codecov.yml` is doing?

**Comment by EdJeeOnGitHub** [comment]
This might help break down GitHub Actions:
![image](https://user-images.githubusercontent.com/34009555/71491900-4d12bd80-282b-11ea-9c62-dcc9017d33fc.png)


We borrowed heavily from here: https://github.com/r-lib/actions/tree/master/examples.

Travis is running `codecov.yml` which assesses how much of the code is tested vs untested (and therefore incidentally runs the tests).

Also @evanjflack  we need to update the badges to link to travis-ci.com since we've migrated - the badges will still work but clicking through the link leads to another redirect.
